iocore/net/SSLSessionCache.h

```diff
@@ -198,3 +201,19 @@ class SSLSessionCache
   SSLSessionBucket *session_bucket = nullptr;
   size_t nbuckets;
 };
+
+class SSLOriginSessionCache
+{
+public:
+  SSLOriginSessionCache();
+  ~SSLOriginSessionCache();
+
+  void insertSession(std::string lookup_key, SSL_SESSION *sess);
+  SSL_SESSION *getSession(std::string lookup_key);
```

Comment:
Please follow the coding style of methods
https://cwiki.apache.org/confluence/display/TS/Coding+Style#CodingStyle-Methods

Reply:
I was trying to keep it consistent with the older code style in the same file, probably should update those in a future PR then.